plugins/ConnectedLivingSpace/CLSAddon.cs

```diff
@@ -17,7 +17,9 @@ public class CLSAddon : MonoBehaviour , ICLSAddon
         private Vector2 scrollViewer = Vector2.zero;
         
         private CLSVessel vessel = null;
-        private int selectedSpace = -1;
+
+        // this var is now restricted to use by the CLS window.  Highlighting will be handled by part.
+        int WindowSelectedSpace = -1;

```

Comment:
Modified Space selection to segregate it from highlighting.  This allows third party highlighting without affecting selected spaces from the CLS window.